config/settings.yml

```diff
@@ -6,5 +6,6 @@ doorkeeper:
   administrators:
     - rainbowdash
 secret_key_base: CHANGE ME
+sentry_dsn: hahano
```

Comment:
Shouldn't this not be set by default? The configs are merged, so if in config/settings/production.yml I set it to `nil`, it will still be set to `hahano`.